src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_PurchaseTabScreen/TicketTabNav_PurchaseTabScreen.tsx

```diff
@@ -117,18 +122,33 @@ export const TicketTabNav_PurchaseTabScreen = ({navigation}: Props) => {
     analytics.logEvent('Ticketing', 'Recently used fare product selected', {
       type: fareProductTypeConfig.type,
     });
+    const getPlace = (
+      harbor: string | undefined,
```

Comment:
Regarding naming, I think it is too early to name it `harbor` here. There is nothing in the domain model that specifies that `fromPlace`/`toPlace` on `pointToPointValidity` are harbors.